domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/timeconversionutils/TimeConversionUtils.java

```diff
@@ -0,0 +1,99 @@
+/*
+ *  Copyright (c) 2020. MobilityData IO.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity.timeconversionutils;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Utility class to carry out operations related to GTFS TYPE type
+ */
+public class TimeConversionUtils {
+    final static String pattern = "([0-9][0-9]|[0-9]):[0-5][0-9]:[0-5][0-9]";
```

Comment:
Yes, instead of calling `String.matches()`, you should compile the regex once and re-use it as a static variable. Here's an example in the timestamp utility class in the GTFS-RT validator:
https://github.com/CUTR-at-USF/gtfs-realtime-validator/blob/master/gtfs-realtime-validator-lib/src/main/java/edu/usf/cutr/gtfsrtvalidator/lib/util/TimestampUtils.java#L44